api/src/api/users/user_routes.py

```diff
@@ -34,3 +39,26 @@ def user_token(x_oauth_login_gov: dict) -> response.ApiResponse:
     logger.info(message)
 
     raise_flask_error(400, message)
+
+
+@user_blueprint.post("/token/logout")
+@user_blueprint.output(UserTokenLogoutResponseSchema)
+@user_blueprint.doc(responses=[200, 401])
+@user_blueprint.auth_required(api_jwt_auth)
+@flask_db.with_db_session()
+def user_token_logout(db_session: db.Session) -> response.ApiResponse:
+    logger.info("POST /v1/users/token/logout")
+
+    user_token_session: UserTokenSession = api_jwt_auth.current_user  # type: ignore
+    with db_session.begin():
+        user_token_session.is_valid = False
+        db_session.add(user_token_session)
+
+    logger.info(
+        "Logged out a user",
+        extra={
+            "user_token_session.user_id": str(user_token_session.user_id),
+        },
```

Comment:
I'd add the token user_token_session.token_id as well so we know which token was logged out.

Reply:
Added to log